addon-resizer/nanny/estimator.go

```diff
@@ -98,6 +98,8 @@ func calculateResources(numNodes uint64, resources []Resource) *corev1.ResourceR
 		overhead := computeResourceOverheadValueString(numNodes, r)
 		newRes := r.Base
 		newRes.Add(resource.MustParse(overhead))
+		// Force initialization of newRes.s so that it could be logged
+		newRes.String()
```

Comment:
Seriously? You really need?
How this is logged?

Reply:
Without this call this value can be logged as: 
```{i:{value:0 scale:0} d:{Dec:0xc4202b5d70} s: Format:DecimalSI}```
WIth this call this becomes:
```{i:{value:0 scale:0} d:{Dec:0xc4202b5d70} s:455m Format:DecimalSI}```

Reply:
How do you print it? If you would log it with:
`klog.("%s", <your var>)`
if would work fine (as this is calling String() method.

Reply:
And it seems like a better way to print it - things like `d:{Dec:0xc4202b5d70}` aren't particularly useful :)

Reply:
This value is not printed directly but instead it is nested in k8s.io/api/core/v1/ResourceRequirements which is printed with `%+v`.
I will change it in a way that I will serialize ResourceRequirements to json before displaying it.